### PR TITLE
Pr 909

### PR DESCRIPTION
**Pull request by krzyzacy**
Add to #909 Not the best way... but seems zero will serialize out becomes default value...

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/test-infra/923)

### CODE REVIEW

**Comment by krzyzacy** [comment]
Remake the commits since I cannot get out of the rebase hell..

**Comment by krzyzacy** [comment]
JESUS I AM NOT MEANT TO DO THIS

**Comment by spxtr** [comment]
Does this work? Is it ready for review?

**Comment by krzyzacy** [comment]
compare [https://b134447156-dev-senlu-dot-sen-lu-test.appspot.com/google-gke#gci-gke-reboot-release-1.3](https://b134447156-dev-senlu-dot-sen-lu-test.appspot.com/google-gke#gci-gke-reboot-release-1.3) and [http://k8s-testgrid.appspot.com/google-gke#gci-gke-reboot-release-1.3](http://k8s-testgrid.appspot.com/google-gke#gci-gke-reboot-release-1.3) ------ however seems if the table is cached it would not get refreshed

File: testgrid/config/config.yaml [hunk 1]
```diff
@@ -17,6 +17,8 @@ default_test_group:
   - configuration_value: Commit # Shows the commit number on column header
   use_kubernetes_client: true # These two fields are depricated and should always be true
   is_external: true
+  alert_stale_results_hours: 0 # Deprecated, set to 0.
+  num_failures_to_alert: 65535 # Deprecated, set to a large enough number to disable cs. Need to make it to work later.
```
Comment (spxtr) [inline testgrid/config/config.yaml]: Maybe make an issue or link to the current one that tracks this?

**Comment by spxtr** [comment]
Looks fine, but it's not clear if this is a stopgap or permanent.

**Comment by krzyzacy** [comment]
It's a temporary solution, I later will remove that alert limit and make code search actually work. Erick recommended [https://sourcegraph.com/](https://sourcegraph.com/). There is an internal ticket for this issue already.

**Comment by spxtr** [comment]
OK sg

**Pull request merged**